src/plugins/fs/vfat.c

```diff
@@ -18,11 +18,11 @@
  */
 
 #include <blockdev/utils.h>
-#include <blockdev/part_err.h>
 #include <check_deps.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdio.h>
```

Comment:
I don't think this is needed (probably a remnant from some debug printing)

Reply:
No, this is intentional, we need `stdio` for `sscanf`. I guess we forgot to include it because it was brought in be parted.